docs/reference/installation.rst

```diff
@@ -83,29 +86,23 @@ If you want to use the API, you also need ``friendsofsymfony/rest-bundle`` and `
                         #ApplicationSonataNewsBundle: ~
                         SonataNewsBundle: ~
 
-
-* Import the ``sonata_news.yml`` file and enable json type for doctrine:
+Register JsonType
```

Comment:
is this still needed?

Reply:
Should be replace with a constraint on Doctrine. And we should deprecate this in favor of `vendor/doctrine/dbal/lib/Doctrine/DBAL/Types/JsonArrayType.php`